workflow/executor/executor.go

```diff
@@ -193,11 +196,19 @@ func (we *WorkflowExecutor) LoadArtifacts() error {
 			if err != nil {
 				return err
 			}
+
+			isZip, err = isZipFile(tempArtPath)
```

Comment:
Not sure we should auto-detect zips, this will break existing workflows that have an explicit step.

`art.GetArchive().Zip != nil`

Reply:
done